packages/ui/src/components/form-input/components/form-textarea.tsx

```diff
@@ -0,0 +1,37 @@
+import { forwardRef, useRef } from 'react'
+import { Controller, useFormContext } from 'react-hook-form'
+
+import { Textarea, TextareaProps } from '@components/form-primitives'
+import { useMergeRefs } from '@utils/mergeUtils'
+
+interface FormTextareaPropsType extends TextareaProps {
+  name: string
+}
+
+const FormTextarea = forwardRef<HTMLTextAreaElement, FormTextareaPropsType>((props, ref) => {
+  const inputRef = useRef<HTMLTextAreaElement | null>(null)
```

Comment:
Do we need `inputRef` here? It is present in my implementation of `form-text-input.tsx` as well. It can be removed.

Reply:
So you suggest remove it from both - TextInput and Textarea?

Reply:
Fixed @praneshg239